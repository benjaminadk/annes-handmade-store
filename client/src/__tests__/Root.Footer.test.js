import React from 'react'
import Footer from '../components/Root.Footer'
import { shallow } from 'enzyme'

test('Root Footer Component renders correctly', () => {
  const wrapper = shallow(<Footer />)
  expect(wrapper).toMatchSnapshot()
})
