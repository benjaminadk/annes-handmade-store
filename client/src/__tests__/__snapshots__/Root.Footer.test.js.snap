// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Footer Component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(Footer) />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "classes": Object {
        "container": "Footer-container-2",
        "link": "Footer-link-3",
        "root": "Footer-root-1",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "classes": Object {
          "container": "Footer-container-2",
          "link": "Footer-link-3",
          "root": "Footer-root-1",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
